docs/cryptodoc/src/05_07_dilithium.rst

```diff
@@ -50,21 +50,21 @@ An overview of the components is provided in Table :ref:`Dilithium components an
 
 .. table::  Dilithium components and file locations.
 
```

Comment:
Arbitrary code location. 
For dilithium we have no full link within the text, like for the other algorithms. Something like:
```
Botan implements the CRYSTALS-Dilithium signature algorithm in
:srcref:`src/lib/pubkey/dilithium/`.
```